src/callbacks_step/analysis.jl

```diff
@@ -470,6 +470,21 @@ pretty_form_utf(quantity) = get_name(quantity)
 pretty_form_ascii(quantity) = get_name(quantity)
 
 
+# Special analyze for `SemidiscretizationHyperbolicParabolic` such that
+# precomputed gradients are available. For now only implemented for the `enstrophy`
+#!!! warning "Experimental code"
+#    This code is experimental and may be changed or removed in any future release.
+function analyze(quantity::typeof(enstrophy), du, u, t, semi::SemidiscretizationHyperbolicParabolic)
```

Comment:
I added the specific `quantity` and `semi` type that this should dispatch on for "safety". I haven't explored this much further but I have a feeling that just dispatching of the `semi` type could cause issues.

Reply:
When I remove the specific `quantity` from the new `analyze` routine it throws an error because there are dispatching issues to compute the `entropy_timederivative`.
```julia
ERROR: LoadError: MethodError: no method matching entropy_timederivative(::SVector{5, Float64}, ::Tuple{SVector{5, Float64}, SVector{5, Float64}, SVector{5, Float64}}, ::CompressibleNavierStokesDiffusion3D{GradientVariablesPrimitive, Float64, CompressibleEulerEquations3D{Float64}})
```
I believe it is because the `enstropy` should use the newly written analyze routine that includes the parabolic cache whereas the `entropy_timederivative` can use the original version of analyze that dispatches on `AbstractSemidiscretization`.

Reply:
Everything is still experimental, isn't it? If so, I propose to proceed with the current status quo and track this in an issue.

Reply:
Sounds good to me 👍

Reply:
This the last remaining open convo. IMHO if this is closed (issue creation was suggested by @ranocha), we can proceed with merging...

Reply:
I made a new issue #1347

Reply:
Great, thanks! /r